ApiReview/API.Polly.Core/NoDocs/Polly/PredicateResult.cs

```diff
@@ -0,0 +1,11 @@
+// Assembly 'Polly.Core'
+
+using System.Threading.Tasks;
+
+namespace Polly;
+
+public static class PredicateResult
```

Comment:
from what I understand this is a factory like type which returns new instances on every call - because they are ValueTasks

Returning new instances from getter is considered bad practice.
I quickly googled to prove my point and found this gorgeous explanation https://stackoverflow.com/questions/2101646/is-object-creation-in-getters-bad-practice. see the accepted answer explaining the best practices compiled from a couple of well known champions

my proposal is to change the properties to methods, ie. `PredicateResult.True()`

Reply:
I don't use ValueTask often so maybe there's something I'm missing that means this won't work, but why don't we just return one we've already created?

```csharp
public static class PredicateResult
{
    public static ValueTask<bool> True { get; } = new(true);
    public static ValueTask<bool> False { get; } = new(false);
}
```

Reply:
OK, then yes in that case let's change it to a method.